src/plugins/googledrive/admin/src/api/getWel.js

```diff
@@ -0,0 +1,16 @@
+import axiosInstance from '../utils/axiosInst';
+export const getWel = async () => {
+  try {
+    const myMessage = await axiosInstance
+      .get('/googledrive/', {
+        //headers: {
```

Comment:
Why don't we need the authentication token? I thought this is a protected endpoint.